server/src/main/scala/net/wiringbits/apis/EmailApi.scala

```diff
@@ -0,0 +1,51 @@
+package net.wiringbits.apis
+
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.simpleemail.AmazonSimpleEmailServiceClientBuilder
+import com.amazonaws.services.simpleemail.model.*
+import net.wiringbits.apis.models.EmailRequest
+import net.wiringbits.config.EmailConfig
+
+import javax.inject.Inject
+
+class EmailApi @Inject() (
+    emailConfig: EmailConfig
+) {
+
+  /* To send mails to unverified accounts you need to configure aws to production mode.
+     In sandbox mode you can only send emails to verified accounts
+      to configure your aws credentials you need to add it at ~/.aws/credentials with
+      [default]
+        aws_access_key_id = ...
+        aws_secret_access_key = ...
+   */
+  def sendEmail(emailRequest: EmailRequest): Unit = {
+    val from = emailConfig.senderAddress
+
+    val htmlBody =
+      s"""
+      <h1>WIRING BITS</h1>
+      <p>${emailRequest.message.body}</p>""".stripMargin
+
+    try {
```

Comment:
The way to do this would be:

```
def unsafe = try {...}

Future {
  blocking(unsafe)
}
```

This is because we should use non-blocking operations.